website/src/categories/components/form/accessibility.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Form accessibility
+tags: accessibility
+eleventyNavigation:
+  parent: Form
+  key: FormAccessibility
+---
+
+<section>
+
+Lorem
```

Comment:
I think we need some information here instead of `Lorem` if we want to show it live on the website.